documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +145,37 @@ After receiving the "Append Setup" reply, it performs the following:
 While this is happening, the server will be periodically sending it `DataAppended` replies acking messages. Note that there can be multiple "Appends Setup" for a given TCP connection. This allows a client to share a connection when producing to multiple Segments.
 
 A client can optimize its appending by specifying a large value in it's `AppendBlock` message, as the events inside of the block do not need to be processed individually.
+
+## Segment Attribute API
+
+| **API**      | **Description**     |
+|-------------|----------|
+|`GetSegmentAttribute`| The requested list of Segment attributes.|
+|`UpdateSegmentAttribute`|  - `newValue`: (Long). |
```

Comment:
Why is the format for this section different? The others all have a section for each message and a table of the fields. This seems to have a table for the messages and a column for fields.

Reply:
@tkaitchuck As  there are many messages....the document will grow longer hence, just mentioning the important messages and its description.